src/main/java/com/arangodb/entity/arangosearch/analyzer/DelimiterAnalyzerProperties.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * DISCLAIMER
+ *
+ * Copyright 2016 ArangoDB GmbH, Cologne, Germany
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright holder is ArangoDB GmbH, Cologne, Germany
+ */
+
+package com.arangodb.entity.arangosearch.analyzer;
+
+
+import java.util.Objects;
+
+/**
+ * @author Michele Rastelli
+ */
+public class DelimiterAnalyzerProperties {
+
+    /**
+     * TODO: clarify the following:
+     * > delimiter (string): the delimiting character(s)
+     * https://www.arangodb.com/docs/3.7/arangosearch-analyzers.html#delimiter
+     * can this be optionally a list? or comma separated chars?
```

Comment:
No, it does not accept an array of strings, nor can multiple characters be specified as single string. You can provide a string longer than a single character however. Only where the full delimiter occurs in the input, there the input gets tokenized. Arangosh example:

```js
require("@arangodb/analyzers").save("csv", "delimiter", {delimiter: ",;\t"})
db._query('RETURN TOKENS("lorem, ipsum,sit;dolor\tamet,;\tconsectetur", "csv")
```

```json
[
  [
    "lorem, ipsum,sit;dolor\tamet",
    "consectetur"
  ]
]
```
```